### ISSUE TITLE
Node not found, comfyui startup shows no errors

### ISSUE THREAD

**Issue by GraftingRayman** [comfyanonymous/ComfyUI#5180]
### Your question

The following node has been working, recently did an upgrade of comfyui and it does not recognise the node
No errors shown in the startup regarding the node

Below is the code for the node, i tried renaming it but fails to show up in the node list, any previously working workflows show this as a missing node. Nothing on my side has been changed. 

Anyone got any ideas?

class GRFlipTileRedRing:
    def __init__(self):
        pass

    @classmethod
    def INPUT_TYPES(cls):
        popular_colors = [
            "black", "white", "red", "blue", "green", "purple", "yellow"
        ]
        return {
            "required": {
                "image": ("IMAGE",),
                "columns": ("INT", {"min": 1}),
                "rows": ("INT", {"min": 1}),
                "border": ("INT", {"min": 0, "default": 0}),
                "colour": (popular_colors,),
                "border_thickness": ("INT", {"min": 1, "default": 5}),
                "seed": ("INT", {"default": None}),  # Adding seed input
            },
            "optional": {
                "flipped_tile_image": ("IMAGE",)
            }
        }

    RETURN_TYPES = ("IMAGE", "IMAGE")
    FUNCTION = "GRRedRingtile_image"
    CATEGORY = "GraftingRayman/Tiles"

    def GRRedRingtile_image(self, image, rows, columns, colour, border=0, border_thickness=5, seed=None, flipped_tile_image=None):
        batch_size, orig_height, orig_width, channels = image.size()

        if seed is not None:
            random.seed(seed)  # Set the seed if provided

        if border > 0:
            orig_height_with_border = orig_height + 2 * border
            orig_width_with_border = orig_width + 2 * border

            bordered_image = torch.ones((batch_size, orig_height_with_border, orig_width_with_border, channels), dtype=image.dtype, device=image.device)
            bordered_image[:, border:-border, border:-border, :] = image
            border_color = self.get_colour_value(colour)
            bordered_image[:, :border, :, :] = border_color  # Top border
            bordered_image[:, -border:, :, :] = border_color  # Bottom border
            bordered_image[:, :, :border, :] = border_color  # Left border
            bordered_image[:, :, -border:, :] = border_color  # Right border
        else:
            bordered_image = image

        new_height = rows * orig_height_with_border if border > 0 else rows * orig_height
        new_width = columns * orig_width_with_border if border > 0 else columns * orig_width
        new_image = torch.zeros((batch_size, new_height, new_width, channels), dtype=image.dtype, device=image.device)
        red_bordered_image = torch.clone(new_image)

        num_tiles_height = (new_height + orig_height_with_border - 1) // orig_height_with_border
        num_tiles_width = (new_width + orig_width_with_border - 1) // orig_width_with_border

        flip_tile = (random.randint(0, rows - 1), random.randint(0, columns - 1))

        for i in range(rows):
            for j in range(columns):
                y_start = i * orig_height_with_border
                y_end = min(y_start + orig_height_with_border, new_height)
                x_start = j * orig_width_with_border
                x_end = min(x_start + orig_width_with_border, new_width)

                tile = bordered_image[:, :y_end - y_start, :x_end - x_start, :]

                if (i, j) == flip_tile:
                    if flipped_tile_image is not None:
                        flipped_tile = flipped_tile_image
                    else:
                        flipped_tile = torch.flip(tile, dims=[2])
                    
                    new_image[:, y_start:y_end, x_start:x_end, :] = flipped_tile

                    red_bordered_image[:, y_start:y_end, x_start:x_end, :] = flipped_tile
                    red_border = self.get_colour_value("red")

                    # Apply red border to the flipped tile
                    red_bordered_image[:, y_start:y_start+border_thickness, x_start:x_end, :] = red_border
                    red_bordered_image[:, y_end-border_thickness:y_end, x_start:x_end, :] = red_border
                    red_bordered_image[:, y_start:y_end, x_start:x_start+border_thickness, :] = red_border
                    red_bordered_image[:, y_start:y_end, x_end-border_thickness:x_end, :] = red_border

                else:
                    new_image[:, y_start:y_end, x_start:x_end, :] = tile
                    red_bordered_image[:, y_start:y_end, x_start:x_end, :] = tile

        return (new_image, red_bordered_image)

    def get_colour_value(self, colour):
        # Map color names to RGB values
        color_map = {
            "black": [0, 0, 0],
            "white": [255, 255, 255],
            "red": [255, 0, 0],
            "blue": [0, 0, 255],
            "green": [0, 255, 0],
            "purple": [128, 0, 128],
            "yellow": [255, 255, 0],
        }
        return torch.tensor(color_map[colour], dtype=torch.float32)

**Comment by GraftingRayman** [comment]
Rolled back to 0.2.2 ComfyUI: 2356[9f291d](2024-07-11) and all the nodes load fine, something has changed preventing some of the nodes from loading.

**Comment by christian-byrne** [comment]
>             "seed": ("INT", {"default": None}),  # Adding seed input

Change `None` to be an int

**Comment by GraftingRayman** [comment]
@christian-byrne thank you, that solved it
